apps/backoffice-v2/src/common/components/organisms/Header/Header.Navbar.tsx

```diff
@@ -86,7 +86,7 @@ export const Navbar: FunctionComponent = () => {
                   className={ctw(
                     `flex items-center gap-x-1 px-1.5 py-1 text-sm font-semibold capitalize text-[#8990AC] hover:bg-[#EBEEF9] hover:text-[#5E688E] active:bg-[#e0e4f6]`,
                     {
-                      'bg-white text-[#20232E]': navItem.filterId === filterId,
+                      'bg-white text-[#20232E]': navItem.filterId === filterId || (navItem.key === 'nav-item-Home' && navItem.href === window.location.pathname),
```

Comment:
The fix should work for any item without children and not use the item's key. Note since we're in React and using React router here you'll use the pathname from `useLocation` and not from the window directly. 🙂